crates/starknet/src/starknet/dump.rs

```diff
@@ -176,6 +191,9 @@ impl Starknet {
                     }
 
                     Ok(transactions)
+                } else if custom_path.is_some() {
+                    // in case of load from http endpoint return error
```

Comment:
this same comment is also present above so maybe redundant?

Reply:
yes not it is, thanks :D

Reply:
removed in c24c7dd6e52159f12b71fb47787dfedef2cfd37a